### ISSUE TITLE
Progress bar inside the quiz

### ISSUE THREAD

**Issue by cMikolai** [chingu-voyages/moonshot-chingu-quiz#60]
As a user, I would like to have an insight of where in the quiz I am and how much of it is left.

E.g. Question 1/10

It would be nice to have it visible as a progress bar.

**Comment by ZumDeWald** [comment]
**Note:** This feature will require that the Quiz Page is in place before starting.
